radis/lbl/calc.py

```diff
@@ -48,7 +48,7 @@ def calc_spectrum(
     molecule=None,
     isotope="all",
     mole_fraction=1,
-    diluent={},
+    diluent=None,
```

Comment:
Should be "air"

Reply:
Done. Now we can add a single diluent as string in the `diluent` argument, by default `diluent='air'`, or can also be `diluent='CO2'`, or any other molecule. In this case we will automatically calculate the molefraction of diluent using 1-molefraction. In other cases user will be required to pass on the diluent as dictionary with it's molefraction.